manifests/webclient.pp

```diff
@@ -31,6 +32,23 @@
     content => template('i2b2/webclient_config_data.js.erb'),
   }
 
+  file { "$webclient_dir/proxy.php" :
+    ensure  => file,
+    content => template('i2b2/proxy.php.erb'),
+  }
+
+  file { "$intermediate_dir/css_declarations.xml":
+    ensure  => file,
+    content => template('i2b2/css_declarations.erb'),
+    require => Exec[ "extract-$webclient_zip" ],
+  }
+  ~>
+  exec { 'insert-css' :
+    cwd     => $webclient_dir,
+    command => "sed -i -e '/<\\/head>/r $intermediate_dir/css_declarations.xml' -e //N default.htm",
```

Comment:
Easiest I can see would be either to modify the regex or to reextract that particular file, none of which seem particularly difficult at first glance.